src/aio/disk_engine.cpp

```diff
@@ -151,10 +157,7 @@ aio_task *disk_file::on_write_completed(aio_task *wk, void *ctx, error_code err,
 disk_engine::disk_engine()
 {
     aio_provider *provider = utils::factory_store<aio_provider>::create(
-        FLAGS_aio_factory_name, dsn::PROVIDER_TYPE_MAIN, this);
-    if (nullptr == provider) {
-        dassert_f(false, "The config value of aio_factory_name is invalid");
-    }
+        native_aio_provider, dsn::PROVIDER_TYPE_MAIN, this);
```

Comment:
So we don't need config `aio_factory_name` now?

Reply:
Yes, of course. I will delete it in the later